src/plugins/auto/auto_schedule.cpp

```diff
@@ -120,33 +120,36 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
     if (_autoSContext->_modelPath.empty())
         _loadContext[ACTUALDEVICE].networkPrecision = GetNetworkPrecision(_autoSContext->_network);
     _loadContext[ACTUALDEVICE].metaDevices = _autoSContext->_devicePriorities;
+    bool isActualDevCPU = false;
     if (isCumulative) {
         std::list<DeviceInformation> validDevices =
             _autoSContext->_plugin->GetValidDevice(_autoSContext->_devicePriorities,
                                                    _loadContext[ACTUALDEVICE].networkPrecision);
-
-        std::string deviceName = "MULTI:";
+        bool isCumuSingleDev = validDevices.size() == 1 ? true : false;
+        std::string deviceName = isCumuSingleDev ? "" : _autoCumuPrefix;
```

Comment:
looks implementation a little bit complicated.
can we :
1) is Cumulative and validDevice size >1, go through original  cumulative logic
2) otherwise, go through original logic and update the hint from cumulative tput to tput
all other logic can remain unchanged.

Reply:
Done